docs/manual/developer/08_content_tests.md

```diff
@@ -15,9 +15,44 @@ To add a script to be checked with mypy use the `mypy_test` macro in `tests/CMak
 
 ### Unit Tests
 
-`ctest` is used to run the unit tests for the `ssg` python package that is in the repo.
+`ctest` is used to run the unit tests for the `ssg` python package that is in the repo. But you need to build content and `ctest` isn't able to run unit tests with differed versions of python. To run only unit tests with differed version of python for the `ssg` python package is recommended to use `tox`.
```

Comment:
This is against the Style Guide https://github.com/ComplianceAsCode/content/blob/master/docs/manual/developer/04_style_guide.md#markup-languages which says that we shall have one sentence per line.